users.json

```diff
@@ -0,0 +1,56 @@
+{
```

Comment:
We spoke about this and the idea was split these 2 elements to get alone the fd token because of the git ignore.